.scrutinizer.yml

```diff
@@ -0,0 +1,18 @@
+filter:
+    paths: [src/*]
+    excluded_paths: [vendor/*, test/*]
```

Comment:
As Jeremy pointed out, should we exclude test directory - tests are important code too.